docs/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
No make available offline banner on the mobile device ( Android + Chrome browser).